src/loader-components/_ExampleTemplate.jsx

```diff
@@ -27,18 +31,48 @@ const propTypes = {
   description: PropTypes.node,
 
   isExpanded: PropTypes.bool,
+
+  isCssExpanded: PropTypes.bool,
 };
 
 const defaultProps = {
   isExpanded: false,
+  isCssExpanded: false,
```

Comment:
lets not add this prop

Reply:
To give my reasoning, JS is a non optional part of the loaded component, CSS is and may not be available for all examples. Additionally if I could go back in time  isExpanded might take a string key ('css', 'js') but changing that at this point would be non-passive at this point and so i think the easiest path forward is just allowing js to be expanded by default.

Reply:
I removed isCssExpanded as a passable prop here: [2e28aa6](https://github.com/cerner/terra-dev-site/pull/281/commits/2e28aa6167c3cf422dbd74674a56591b9ecbdf9b), instead I just redefined it inside the ExampleTemplate itself.